odl/space/ntuples.py

```diff
@@ -132,6 +132,30 @@ def element(self, inp=None, data_ptr=None):
             else:
                 raise ValueError('Cannot provide both `inp` and `data_ptr`')
 
+    def zero(self):
+        """Create a vector of zeros.
+
+        Examples
+        --------
+        >>> c3 = Cn(3)
+        >>> x = c3.zero()
+        >>> x
+        Cn(3).element([0j, 0j, 0j])
+        """
+        return self.element(np.zeros(self.size, dtype=self.dtype))
+
+    def one(self):
+        """Create a vector of ones.
+
+        Examples
+        --------
+        >>> c3 = Cn(3)
+        >>> x = c3.one()
+        >>> x
+        Cn(3).element([(1+0j), (1+0j), (1+0j)])
+        """
+        return self.element(np.ones(self.size, dtype=self.dtype))
```

Comment:
Well, yeah I'll agree it is garbage, but if you insist on calling them, why not? At least the `zero` guy returns something remotely decent:

```
>>> np.zeros(5, dtype='U1')
array([u'', u'', u'', u'', u''], 
      dtype='<U1')
```